resources/scripts/generate-build-data.sh

```diff
@@ -461,6 +500,7 @@ fetchAndPrepareTestsInfo "${TESTS_INFO}" "${BO_BUILD_URL}/tests/?limit=10000000"
 fetchAndPrepareTestSummaryReport "${TESTS_SUMMARY}" "${BO_BUILD_URL}/blueTestSummary/" "test_summary" "${DEFAULT_LIST}" "${TESTS_INFO}"
 fetchAndPrepareTestCoverageReport "${TESTS_COVERAGE}" "${BUILD_URL}" "test_coverage" "${DEFAULT_HASH}"
 fetchAndPrepareBuildInfo "${BUILD_INFO}" "${BO_BUILD_URL}/" "build" "${DEFAULT_HASH}"
+prepareBuildAnalysis "analysis"
```

Comment:
Wdyt about a more descriptive key, something like `errorAnalysis`, or `buildErrors`?
```suggestion
prepareBuildAnalysis "buildErrors"
```

Reply:
I'm somehow skeptical to use `buildErrors` since it does not imply a build error by definition. 

For instance, when I tested the script locally with:

```
https://apm-ci.elastic.co/blue/rest/organizations/jenkins/pipelines/apm-agent-nodejs/apm-agent-nodejs-mbp/master/runs/1151 SUCCESS 5794469
```

The build finished correctly, but there were a couple of `interruptedException`:

```
INFO: curl https://apm-ci.elastic.co/blue/rest/organizations/jenkins/pipelines/apm-agent-nodejs/apm-agent-nodejs-mbp/master/runs/1151/ -o build-info.json
+ echo 'INFO: prepareBuildAnalysis'
INFO: prepareBuildAnalysis
+ '[' -e pipeline-log.txt ']'
++ grep -i 'cannot contact .*InterruptedException' --count pipeline-log.txt
+ interruptedException=2
```

So it might not be the correct name, maybe `errorMetrics`?

Reply:
I like error metrics much more 👏 as it does not force readers to know the context.